applications/FluidDynamicsApplication/custom_conditions/navier_stokes_wall_condition.h

```diff
@@ -78,13 +78,15 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) NavierStokesWallCondition : public
     struct ConditionDataStruct
     {
         double wGauss;                                  // Gauss point weight
-        bool OutletInflowPreventionSwitch;              // Outlet inflow (i.e. backflow) prevention switch
-        double charVel;                                 // Problem characteristic velocity (used in the outlet inflow prevention)
         array_1d<double, 3> Normal;                     // Condition normal
         array_1d<double, TNumNodes> N;                  // Gauss point shape functions values
         Vector ViscousStress;                           // Viscous stresses that are retrieved from parent
     };
 
+    static constexpr std::size_t VoigtSize = 3 * (TDim-1);
+    static constexpr std::size_t BlockSize = TNumNodes + 1;
```

Comment:
```suggestion
    static constexpr std::size_t BlockSize = TDim + 1;
```

Shouldn't it be like this?

Reply:
It should. Thanks!